soccer/config_server/config_server.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
```

Comment:
Not unique to this file, but the swap from ".hpp" -> ".h", is that some style thing?